tests/parametric/test_trace_sampling.py

```diff
@@ -147,6 +147,75 @@ def test_trace_sampled_by_trace_sampling_rule_glob_match(self, test_agent, test_
         assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == 2
         assert span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == 1.0
 
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_TRACE_SAMPLE_RATE": 0,
+                "DD_TRACE_SAMPLING_RULES_FORMAT": "glob",
+                "DD_TRACE_SAMPLING_RULES": json.dumps(
+                    [
+                        {"name": "wEb.rEquEst", "sample_rate": 1},
+                    ]
+                ),
+            },
+            {
+                "DD_TRACE_SAMPLE_RATE": 0,
+                "DD_TRACE_SAMPLING_RULES": json.dumps([{"service": "wEbSerVer", "sample_rate": 1}]),
+            },
+            {
+                "DD_TRACE_SAMPLE_RATE": 0,
+                "DD_TRACE_SAMPLING_RULES_FORMAT": "glob",
+                "DD_TRACE_SAMPLING_RULES": json.dumps([{"resource": "/rAnDom", "sample_rate": 1}]),
+            },
+            {
+                "DD_TRACE_SAMPLE_RATE": 0,
+                "DD_TRACE_SAMPLING_RULES_FORMAT": "glob",
+                "DD_TRACE_SAMPLING_RULES": json.dumps([{"tags": {"key": "vAlUe"}, "sample_rate": 1}]),
+            },
+        ],
+    )
+    @missing_feature(library="cpp", reason="Not implemented")
+    @missing_feature(library="nodejs", reason="Not implemented")
+    def test_field_case_insensitivity(self, test_agent, test_library):
+        """Test span sampling tags are added when a rule with glob patterns with special characters * and ? match"""
+        with test_library:
+            with test_library.dd_start_span(
+                name="web.request", service="webserver", resource="/random", tags=[("key", "value")]
+            ) as span:
+                pass
+
+        span = find_only_span(test_agent.wait_for_num_traces(1))
+
+        assert span["metrics"].get(SAMPLING_PRIORITY_KEY) == 2
+        assert span["metrics"].get(SAMPLING_RULE_PRIORITY_RATE) == 1.0
+
+    @pytest.mark.parametrize(
+        "library_env",
+        [
+            {
+                "DD_TRACE_SAMPLE_RATE": 0,
+                "DD_TRACE_SAMPLING_RULES": json.dumps(
+                    [
+                        {"name": "wEb.*", "sample_rate": 1},
+                    ]
+                ),
+            }
+        ],
+    )
+    def test_trace_sampling_rules_format_not_needed(self, test_agent, test_library):
+        """Glob should work by default without specifying the format"""
```

Comment:
The assertions/spans in this test look identical to test_field_case_insensitivity. S
hould we add the `library_env` in this test to  test_field_case_insensitivity parameterized test? This way we have one test with 5 scenarios.

Reply:
I could just remove it from all the tests since I believe no language currently requires the glob environment variable